### DISCUSSION TITLE
WARN issue when installing.

### DISCUSSION THREAD

**Discussion by zechs-merquise** [tailwindlabs/tailwindcss#8447] [Help]
I followed every step the installation sugested me and still can't see it reflected on HOST.
![image](https://user-images.githubusercontent.com/106116790/170595087-f288418f-fc50-4c08-8207-e13223ec7a1a.png)
![image](https://user-images.githubusercontent.com/106116790/170595122-78ff4c67-0e6c-45fc-add5-4235f8d90e94.png)
![image](https://user-images.githubusercontent.com/106116790/170595150-08ab255e-f12d-48e5-b442-bfa4d2c68b9e.png)
![image](https://user-images.githubusercontent.com/106116790/170595177-a9763317-c804-4959-9dd0-f8968d335275.png)

Someone knows how to correct this issue?
Thanks!

**Comment by zslabs** [comment]
You need to add jsx to the content extensions in the config. You only have html and js, which App.jsx is neither.
